src/gluonnlp/model/bert.py

```diff
@@ -495,7 +504,7 @@ def _decode(self, sequence, masked_positions):
 }
 
 
-def bert_12_768_12(dataset_name=None, vocab=None, pretrained=True, ctx=mx.cpu(),
+def bert_12_768_12(dataset_name=None, bert_vocab=None, pretrained=True, ctx=mx.cpu(),
```

Comment:
Please do not change argument name for public APIs

Reply:
OK, I will fix all problems like this.